dev/integration_tests/ios_add2app/build_and_test.sh

```diff
@@ -16,4 +16,4 @@ popd
 
 pod install
 os_version=$(xcrun --show-sdk-version --sdk iphonesimulator)
-xcodebuild -workspace ios_add2app.xcworkspace -scheme ios_add2appTests -sdk "iphonesimulator$os_version" -destination "OS=$os_version,name=iPhone X" test
\ No newline at end of file
+xcodebuild -workspace ios_add2app.xcworkspace -scheme ios_add2appTests -sdk "iphonesimulator$os_version" -destination "OS=$os_version,name=iPhone X" test | xcpretty && exit ${PIPESTATUS[0]}
```

Comment:
@gaaclarke did something like this which seems like we could land it now and it will just magically start working:

```
PRETTY="cat"
if which xcpretty; then
  PRETTY="xcpretty"
fi

set -o pipefail && xcodebuild -workspace ios_add2app.xcworkspace -scheme ios_add2appTests -sdk "iphonesimulator$os_version" -destination "OS=$os_version,name=iPhone X" test | $PRETY
```

Reply:
(start working once xcpretty is available on the image, which we could also still temporarily do by installing it oursevles)

Reply:
oh, right. Sure

Reply:
unfortunately cat makes a mess of our tool's outputs which makes extensive use of backspace characters but it's not the worst if we're expecting CIs to have xcpretty.

Reply:
You can try `tee` instead of `cat`.